app/components/pagination.hbs

```diff
@@ -0,0 +1,31 @@
+{{#if (gt @meta.page 1)}}
+  <span class="pagination">
+    <LinkTo
+      @route={{@route}}
+      @query={{hash page=(dec @meta.page)}}
+    >
+      {{t "podcasts.pagination_prev"}}
```

Comment:
Could adjust the i18n lang to be a more generic pagination prev/next rather than podcasts specific for this component.